### PR TITLE
EdgeIdentity v5.0.0

### PR DESCRIPTION
**Pull request by emdobrin**
## Description
Updates minimum supported iOS, tvOS to 12.0
Fixes podspec verification failure
CircleCI, workflow Xcode updates
Readme updates

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by kevinlind — APPROVED** [review]
Looks good! For the Codecov build errors, maybe try updating the orb to 4.0.1?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -20,20 +20,23 @@ on:
...
-    runs-on: macos-latest
+    runs-on: macos-13
     steps:
     - uses: actions/checkout@v2
       with:
         ref: main
     - uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: '14.1'
+        xcode-version: '15'
```
Comment (cacheung) [inline .github/workflows/release.yml:30]: 15.0?

File: Script/test-podspec.sh [hunk 2]
```diff
@@ -21,18 +21,34 @@ mkdir -p $PROJECT_NAME && cd $PROJECT_NAME
...
-platform :ios, '11.0'
+platform :ios, '12.0'
 target '$PROJECT_NAME' do
   use_frameworks!
-  pod 'AEPCore', '~> 4.0'
-  pod 'AEPIdentity', '~> 4.0'
-  pod 'AEPServices', '~> 4.0'
-  pod 'AEPRulesEngine', '~> 4.0'
-  pod 'AEPEdge', '~> 4.0'
+  pod 'AEPCore', '~> 5.0'
```
Comment (cacheung) [inline Script/test-podspec.sh:47]: Should we use ~> 5.0.0?

File: Script/test-podspec.sh [hunk 3]
```diff
@@ -64,18 +80,34 @@ rm -rf $PROJECT_NAME
...
-platform :tvos, '11.0'
+platform :tvos, '12.0'
 target '$PROJECT_NAME' do
   use_frameworks!
-  pod 'AEPCore', '~> 4.0'
-  pod 'AEPIdentity', '~> 4.0'
-  pod 'AEPServices', '~> 4.0'
-  pod 'AEPRulesEngine', '~> 4.0'
-  pod 'AEPEdge', '~> 4.0'
+  pod 'AEPCore', '~> 5.0'
```
Comment (cacheung) [inline Script/test-podspec.sh:106]: Should we use ~> 5.0.0?

Comment (emdobrin) [reply Script/test-podspec.sh:106]: Using the optimistic operator here so `~> 5.0` means 5.0 and up until 6.0, but not including 6.0. So 5.0.0 is included.

**Review by cacheung — APPROVED** [review]

**Pull request merged**